internal/version/versions.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 Tetrate
```

Comment:
I'm not sure if this is the right package for this file. I've tried to introduce another package for this interface only: "remote", but felt not right. Please let me know the right location for this. Thank you!

Reply:
I feel this is the right place to put! +1